ui/shared/status/StatusButton.qml

```diff
@@ -69,7 +69,9 @@ Button {
             anchors.left: iconLoader.active ? iconLoader.right : parent.left
             anchors.leftMargin: iconLoader.active ? Style.current.smallPadding : 0
             color: !enabled ? Style.current.buttonDisabledForegroundColor : 
-              (hovered || highlighted) ? Style.current.blue : control.color
+              (hovered || highlighted) ?
+                type === "warn" ? Style.current.danger : Style.current.blue :
```

Comment:
Instead fo changing it here, you could leave it as you had before (`control.color`), and just change the actual color above like so:
```qml
property color color: type === "warn" ? Style.current.danger : Style.current.buttonForegroundColor
```

That way, the color stays configurable even if the type is warn